earn/src/components/boost/CollectFeesWidget.tsx

```diff
@@ -59,9 +73,9 @@ export default function CollectFeesWidget(props: CollectFeesWidgetProps) {
     address: ALOE_II_BOOST_NFT_ADDRESS[activeChain.id],
     abi: boostNftAbi,
     functionName: 'modify',
-    args: [ethers.BigNumber.from(cardInfo.nftTokenId || 0), 1, modifyData, [true, true]],
+    args: [ethers.BigNumber.from(cardInfo.nftTokenId || 0), 1, modifyData, oracleSeed ?? 0],
```

Comment:
Maybe default to Q32, so if something weird happens users aren't screwed on gas.